iosApp/Survey/Sources/Presentation/Modules/SurveySelection/SurveySelectionContainerView.swift

```diff
@@ -29,8 +34,8 @@ struct SurveySelectionContainerView: View {
 
             HStack {
                 Spacer()
-                if isShowingAccountView {
-                    AccountView()
+                if isShowingAccountView, let account = dataSource.viewState.accountUiModel {
```

Comment:
Just an edge-case concern that I realize we are going to show a blank view here when there is not a valid account. I know it is not a part of the requirement but shouldn't we handle to show a fallback UI instead? It might feel a bit weird seeing nothing here over a long period 🙏

Reply:
If there's no `dataSource.viewState.accountUiModel`, there will be no profile image button for user to tap to open this page.

Reply:
So it is expected that in such cases, there is no way for use to logout? 😅

Reply:
@minhnimble in `SurveySelectionViewModel`, `AccountUiModel` is guaranteed to return and user will always be able to logout, unless the app crash.

Reply:
Noted, then I guess we can consider revisit this if the missing data case actually happens. 👌